packages/next/src/views/Edit/Default/index.tsx

```diff
@@ -151,9 +146,7 @@ export const DefaultEditView: React.FC = () => {
       isEditing,
       refreshCookieAsync,
       adminRoute,
-      locale,
```

Comment:
It looks like eslint was not linting for dep arrays. Please merge this branch with latest beta which includes that. Then resave any changed tsx files.

Reply:
@PatrikKozak it still looks like locale is not in the dep array. Eslint should warn you about it and make you fix it.